src/lib/utils/hooks/useTheme.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useContext } from 'react'
```

Comment:
IconSvg plays nicely with styles in JS, but not CSS classes, so it's more natural to determine the color via a theme object, rather than supplying a class name.

Thoughts on this approach, @jay-hodgson @conundrumgirl @yjyoo773 ? Is this going to be a pain to keep in line with SCSS? Should we fully commit to one tech, if so, which one?

A caller (e.g. a portal) could simply wrap the app in 
```tsx
const myPortalTheme = {
	colors: {
		success: "green"
	}
}

<ThemeProvider theme={myPortalTheme>
	// the rest of the app
</ThemeProvider>
```

Reply:
This is how material-ui is doing it. They provide a number of ways of customization including styled components and regular css classes. It appears to play well together.

Reply:
I support this direction if others are on board, since it's a good experience when using MUI (and it seems natural with our use of context and hooks).
Would it be possible to customize each portal via modifying the theme (I think they're scss variable based at the moment)?

Reply:
@jay-hodgson yep! We would just need to export the ThemeContext.Provider or add the ThemeContext to our existing SynapseContext component, and then a portal could supply their own theme overrides.